starter-code/src/functions-and-arrays.js

```diff
@@ -11,13 +14,37 @@ const words = [
   'crackpot'
 ];
 
+var LongestWord = words.sort((a,b)=>b.length - a.length);
```

Comment:
Sur cet exercice, il n'est pas nécessaire d'utiliser sort. Vous avez simplement à vérifier dans votre boucle for si ton word.length est inférieur à chaque élément de ton array (comme vous l'avez fait des lignes 22 à 28)